tests/fee_estimation/test_fee_estimation_rpc.py

```diff
@@ -44,6 +44,26 @@ async def setup_node_and_rpc(
     return client, full_node_rpc_api
 
 
+@pytest_asyncio.fixture(scope="function")
+async def setup_node_and_rpc_no_blocks_farmed(
+    two_wallet_nodes_services: Tuple[List[Service[FullNode]], List[Service[WalletNode]], BlockTools],
```

Comment:
We could avoid setting up two wallets here I think?

Reply:
Ya, agreed